projects/js-packages/components/components/jetpack-footer/index.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import AutomatticBylineLogo from '../automattic-byline-logo';
+import './style.scss';
+
+/**
+ * JetpackFooter component definition.
+ *
+ * @param {object} props - Component properties.
+ * @param {object} props.aboutPageUrl - Link for 'An Automattic Airline'.
+ * @param {object} props.moduleName - Name of the moduel, e.g. 'Jetpack Search'.
+ * @param {object} props.className - additional className of the wrapper, default only: `jp-dashboard-footer`.
+ *
+ * @returns {React.Component} JetpackFooter component.
+ */
+export default function JetpackFooter( {
+	aboutPageUrl,
+	moduleName = 'Jetpack',
```

Comment:
Does this also need to be translated with `__`? 🙂

Reply:
Good point. I did think about it. I thought it's our brand and since wouldn't need to be translated. But a brief search across our code base, found a couple of cases translating `Jetpack`. I guess for most languages it is kept just as is. So added translation in 998e17337